PF1/Assignment1/OctalConverter.java

```diff
@@ -0,0 +1,56 @@
+package assignment1;
```

Comment:
package name should be same. In java file assignment having small caps but package having capital caps